build.gradle

```diff
@@ -99,3 +100,62 @@ wpi.java.configureTestTasks(test)
 tasks.withType(JavaCompile) {
     options.compilerArgs.add '-XDstringConcat=inline'
 }
+
+def getRobotVersionData() { 
+    def robotVersion = new StringBuilder()     
+    robotVersion.append("Commit: @"+"git rev-parse --short HEAD".execute().text.replace('\n', '')+" by "+("git config --list".execute().text).split("user.name=")[1].split("\n")[0]+"\n")       
```

Comment:
The `[1]` is failing here currently in the build... i think this is because this would work on your local machine but the output is differennt when running as a github action. Can we come up with a way to ignore this on github action? Maybe it should only be running on `deploy` and `simulateExternalJavaRelease` and not `build` ?